.github/workflows/test.yaml

```
@@ -134,10 +136,10 @@ jobs:
           # Just Python 3.11 with default settings.
           - python-version: '3.11'
 
-          # Just Python 3.13 alpha with default settings.
+          # Pre-release Python 3.13 with default settings.
           # For now, no dependencies as cffi doesn't work on 3.13.
           # No coverage also.
-          - python-version: '3.13.0-alpha.3'
+          - python-version: '3.13.0-beta.2'
```

Maybe this is a better approach 

[From here](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-a-python-version) 

> It's also possible to use x.y-dev syntax to download and set up the latest patch version of Python, alpha, beta and rc (release candidate) releases included. (for specified major & minor versions):

```suggestion
          - python-version: '3.13-dev'
```

---

Ooh nice.